packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -134,9 +129,11 @@ async function getAndUpdateSidechainConverterKeys(): Promise<string> {
       })
       .promise();
   }
-  if (!activeKey)
+  if (activeKeys.length < 1)
     throw new Error(`No active key found in sidechain keys table - can't do conversion`);
-  return activeKey;
+
+  // pick a random key
+  return activeKeys[Math.floor(Math.random() * activeKeys.length)];
```

Comment:
`.length - 1`, otherwise we can end up w/ a position equal to the array length

Reply:
we can't - example:
`Math.floor(.99 * 50)`
= `Math.floor(49.5)`
= `49`

Reply:
💯 , forgot `.random()` is 
> greater than or equal to 0 and less than 1